src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBase.cpp

```diff
@@ -859,8 +868,161 @@ void CompositionalMultiphaseBase::initializePostInitialConditionsPreSubGroups()
 
   // Initialize primary variables from applied initial conditions
   initializeFluidState( mesh );
+
+  // Compute some statistics on the initial state
+  computeReservoirStatistics( mesh, true );
+
+}
+
+void CompositionalMultiphaseBase::computeStatistics( real64 const & GEOSX_UNUSED_PARAM( time ),
+                                                     real64 const & GEOSX_UNUSED_PARAM( dt ),
+                                                     integer cycleNumber,
+                                                     DomainPartition & domain,
+                                                     bool outputStatisticsToScreen )
+{
+  MeshLevel const & mesh = domain.getMeshBody( 0 ).getMeshLevel( 0 );
+
+  // output the number of Newton iterations if this is the main solver
+  if( outputStatisticsToScreen && m_nonlinearSolverParameters.m_totalSuccessfulNewtonNumIterations > 0 )
+  {
+    GEOSX_LOG_LEVEL_RANK_0( 1, getName()
+                            << ": Total number of time steps = " << cycleNumber+1
+                            << ", successful nonlinear iterations = " << m_nonlinearSolverParameters.m_totalSuccessfulNewtonNumIterations
+                            << ", wasted nonlinear iterations = " << m_nonlinearSolverParameters.m_totalWastedNewtonNumIterations );
+  }
+
+  // output the average pressure, average temperature, pore volumes
+  computeReservoirStatistics( mesh, outputStatisticsToScreen );
```

Comment:
Move this `computeReservoirStatistics( mesh, outputStatisticsToScreen );` into the `if ( outputStatisticsToScreen...` scope (I think).

Reply:
I am no longer going to output anything to terminal (average pressure, nonlinear iterations, etc), so I got rid of `outputStatisticsToScreen`.